docs/Main.md

```diff
@@ -37,6 +37,8 @@ style="max-width:100%;"></a>
 </p>
 \endhtmlonly
 
+![banner](docs/Images/banner.png)
```

Comment:
Should the word "spectre" be a bit larger? It looks a tad small in the space that it's in.

Reply:
🤷‍♂️ I think it's fine, it's personal preference I suppose. Try this size and adjust later if we find it's too small?

FYI, I intend to add a gravitational wave visualization to this banner when I find the time, to have the "s" radiate a burst of GWs in 3D.